libdnf.spec

```diff
@@ -1,10 +1,10 @@
-Name:           libdnf
-Version:        1.0.0
+Name:           libdnf5
```

Comment:
This commit is obviously misnamed, it doesn't just fix anything, it does the renaming of the packages.

Reply:
It does both. It did not build on F33 and F34 at all.

Reply:
If we're not importing this into Koji, then we can just add a `libdnf5` subpackage for parallel installability, while leaving the top-level name still `libdnf`.

You can see I did that with [the Mageia spec](https://download.copr.fedorainfracloud.org/results/ngompa/dnf5-mga/mageia-cauldron-x86_64/01705155-libdnf/libdnf.spec):

```rpm-spec
# ========== libdnf ==========

%package -n libdnf3
Summary:        Package management library
#Requires:       libmodulemd{?_isa} >= {libmodulemd_version}
Requires:       %{_lib}solv1%{?_isa} >= %{libsolv_version}
Requires:       %{_lib}repo0%{?_isa} >= %{librepo_version}

%description -n libdnf3
Package management library.

%files -n libdnf3
%{_libdir}/libdnf.so.3
%license COPYING.md
%license lgpl-2.1.txt

# ========== libdnf ==========
```